src/contracts/IdleRouter.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: GPL-3.0
-pragma solidity 0.8.11;
+pragma solidity ^0.8.11;
```

Comment:
Please leave the pragma locked here.  It's important that when we release this, we are using the same solidity version we have tested against.

Reply:
Should I change the pragma for the other files? Initially I changed it because of the linter warnings